src/pages/portfolio/index.tsx

```diff
@@ -156,9 +194,21 @@ const PortfolioPage: FC<{ data: ProjectGQL }> = ({ data }) => {
         />
         <Box className={classes.mainContent}>
           <Box className={classes.projectsContainer}>
-            <SectionTitle className={classes.title}>My works</SectionTitle>
+            <Box className={classes.navbarBox}>
+              <SectionTitle className={classes.title}>My works</SectionTitle>
+              {!isMobile && (
+                <FilterTabs
+                  className={classes.navbarTitles}
+                  indicatorColor="primary"
+                  textColor="primary"
+                  handleChange={handleChange}
+                  navbarTitle={navbarTitle}
+                  projectLabelsTitles={projectLabelsTitles}
```

Comment:
Here and inside of the component, I also suggest switching the name to `projectLabels`